go.mod

```diff
@@ -1,5 +1,5 @@
 module github.com/vishvananda/netns
 
-go 1.17
+go 1.22
```

Comment:
This should go in a separate PR as it's orthogonal to the primary intent of the PR. Also, please don't bump the `go` version... it means different things in the context of a library than it does in your app.

Reply:
ack will remove this